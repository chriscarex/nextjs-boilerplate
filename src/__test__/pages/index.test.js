import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Index from '../../pages/index';

const setup = () => {
  const component = shallow(<Index />);

  return {
    component
  };
};

describe('Index page', () => {
  it('should match snapshot', () => {
    const { component } = setup();
    expect(toJson(component)).toMatchSnapshot();
  });
});
