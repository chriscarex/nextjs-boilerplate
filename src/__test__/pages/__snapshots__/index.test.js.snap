// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match snapshot 1`] = `<Home />`;
